api-doc.md

```diff
@@ -0,0 +1,301 @@
+            },
+        ]
+
+```
+
+### 2. Get Career By ID
+
+```
+- Method : GET
+- Endpoint : /api/careers/:uniqueId
+- Description : API for get careers by provided id
+- Request:
+    - Headers : none
+    - Body : none
+
+- Response :
+
+    - If API is not found or career data not found then,
+    - Status Code: 404 not found
+    - Data: no response data
+
+    - If any server error is there then,
+    - Status Code: 500 Internal server error
+    - Data: no response data
+
+    - If request will success ,
+    - Status Code: 200 Ok
+    - Headers : none
+    - Data :
+        {
+            "id": 1,
+            "title": "IOS Developer",
+            "summary": "We have an opening for a passionate iOS developer who can develop and maintain applications for iOS devices. As an iOS developer you will be a part of a highly agile team tasked with developing new features. If you love to tackle new challenges, we would love to meet you!",
+                "description": "<h2><span style=\"font-size:18px\"></span><span></span><span style=\"color:#3d3d3d;\"><span style=\"font-size:22px;\"><strong>IOS Developer Description</strong></span></span></h2>",
+            "button_name": "Apply",
+            "qualification": "B.E/B.Tech/BCA/MCA/MSc IT degree in Computer Science, Engineering, or a related subject ",
+            "employment_type": "full_time",
+            "base_salary": 15000,
+            "experience": "0-3 years",
+            "is_active": true,
+            "skills": "",
+            "total_openings": 1,
+            "responsibilities": "",
+            "icon_name": "fab fa-apple",
+            "unique_id": "ios-developer-f81c173e-4098-4e8f-b5d3-3d03ab360571",
+            "seo_title": "iOS developer jobs in surat at Canopas, Mobile developer jobs",
+            "seo_description": "Find iOS developer job in surat. If we hire you as an iOS developer, you will be responsible for designing and coding the base application, ensuring the quality, fixing bugs, maintaining the code, and implementing app updates.",
+            "apply_seo_title": "Apply for iOS developer job in surat or mobile developer jobs in surat at canopas software",
+            "apply_seo_description": "Apply for iOS developer job at Canopas and be part of a dynamic and versatile iOS app development team."
+        }
+
+```
+
+### 3. Send career mail
+
+```
+- Method : POST
+- Endpoint : /api/send-career-mail
+- Description : API for send career mail
+- Request:
+    - Headers : none
+    - Body :
+        {
+            "job_title": "Web Developer",
+            "name": "New Web Developer",
+            "email": "developer@canopas.com",
+            "phone": "9099999999",
+            "place": "surat",
+            "references" : "From canopas",
+            "message"  : I'm a very good programer,
+            "file" : resume.pdf,
+        }
+
+
+- Response :
+
+    - If API is not found or career details data not found then,
```

Comment:
We are not fetching any career details, so here only will be ` If API is not found`